.github/workflows/clippy_bors.yml

```
@@ -0,0 +1,151 @@
+        exclude:
+        - os: ubuntu-latest
+          host: x86_64-apple-darwin
+        - os: ubuntu-latest
+          host: x86_64-pc-windows-msvc
+        - os: macos-latest
+          host: x86_64-unknown-linux-gnu
+        - os: macos-latest
+          host: i686-unknown-linux-gnu
+        - os: macos-latest
+          host: x86_64-pc-windows-msvc
+        - os: windows-latest
+          host: x86_64-unknown-linux-gnu
+        - os: windows-latest
+          host: i686-unknown-linux-gnu
+        - os: windows-latest
+          host: x86_64-apple-darwin
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Debug
+      run: |
+        echo $RUSTUP_HOME
+        echo $CARGO_HOME
+        exit 1
+    - name: Install dependencies (Linux-i686)
+      run: |
+        sudo dpkg --add-architecture i386
+        sudo apt-get update
+        sudo apt-get install gcc-multilib libssl-dev:i386
+      if: matrix.host == 'i686-unknown-linux-gnu'
+    - name: rust-toolchain
+      uses: actions-rs/toolchain@v1.0.3
+      with:
+        toolchain: nightly
+        target: ${{ matrix.host }}
+        profile: minimal
+    - name: Cache cargo dir
+      uses: actions/cache@v1
+      with:
+        path: ~/.cargo
+        key: ${{ runner.os }}-cargo
+    - name: Checkout
+      uses: actions/checkout@v2.0.0
+    - name: Master Toolchain Setup
+      run: bash setup-toolchain.sh
+      env:
+        HOST_TOOLCHAIN: ${{ matrix.host }}
+
+    - name: Set SYSROOT path (Linux)
+      run: |
+        SYSROOT=$(rustc --print sysroot)
+        echo "::set-env name=LD_LIBRARY_PATH::${SYSROOT}/lib${LD_LIBRARY_PATH+:${LD_LIBRARY_PATH}}"
+      if: runner.os == 'Linux'
+    - name: Set SYSROOT path (MacOS)
+      run: |
+        SYSROOT=$(rustc --print sysroot)
+        sudo mkdir -p /usr/local/lib
+        sudo find "$SYSROOT/lib" -maxdepth 1 -name '*dylib' -exec ln -s {} /usr/local/lib \;
+      if: runner.os == 'macOS'
+    # - name: Set SYSROOT path (Windows)
+    #   run: |
+    #     $env:PATH += ';$(rustc --print sysroot)/bin'
+    #     echo "::set-env name=PATH::$env:PATH"
+    #   if: runner.os == 'Windows'
+    # - name: Debug
+    #   run: |
+    #     echo $env:PATH
+    #   if: runner.os == 'Windows'
+    - name: Build
+      run: cargo build --features deny-warnings
+    - name: Test
+      run: cargo test --features deny-warnings
+    - name: Test clippy_lints
+      run: cargo test --features deny-warnings
+      working-directory: clippy_lints
+    - name: Test rustc_tools_util
+      run: cargo test --features deny-warnings
+      working-directory: rustc_tools_util
+    - name: Test clippy_dev
+      run: cargo test --features deny-warnings
+      working-directory: clippy_dev
+    - name: Test cargo-clippy
+      run: ../target/debug/cargo-clippy
+      working-directory: clippy_workspace_tests
+    - name: Test clippy-driver
+      run: |
```

```suggestion
      bash: |
```
Otherwise you get PowerShell on Windows.

---

Ah thanks! That'll help. 

FYI: according to the documentations it's
```yaml
run: ...
shell: bash
```

---

Oops, I've mistaken syntax with different CI.